content/en/blog/performance-benchmarks-gke.md

```diff
@@ -0,0 +1,354 @@
+have mutual anti-affinity, so one node is required per pod.
+
+For example, the node pools that are used in our continuous integration setup
+are configured as follows:
+
+| Pool name            | Node count | Machine type   | Kubernetes labels                          |
+| :------------------- | ---------: | :------------- | :----------------------------------------- |
+| system               |          2 | e2-standard-8  | default-system-pool:true,&nbsp;pool:system |
+| drivers-ci           |          8 | e2-standard-2  | pool:drivers-ci                            |
+| workers-c2-8core-ci  |          8 | c2-standard-8  | pool:workers-c2-8core-ci                   |
+| workers-c2-30core-ci |          8 | c2-standard-30 | pool:workers-c2-30core-ci                  |
+
+Since each scenario in our tests requires one driver and two workers, this
+configuration supports four simultaneous tests on 8-core machines and four on
+30-core machines. Drivers require few resources, and do not have mutual
+anti-affinity. We find it convenient to schedule them on two-core machines with
+a node count set to the required number of drivers, rather than on a larger
+shared machine, because that allows the driver pool to be resized together with
+the worker pools. The controller itself is scheduled in the `system` pool.
+
+In addition to the pools used in continuous integration, our cluster contains
+additional node pools that can be used for ad hoc testing:
+
+| Pool name      | Node count | Machine type   | Kubernetes labels                                 |
+| :------------- | ---------: | :------------- | :------------------------------------------------ |
+| drivers        |          8 | e2-standard-8  | default-driver-pool:true,&nbsp;pool:drivers       |
+| workers-8core  |          8 | e2-standard-8  | default-worker-pool:true,&nbsp;pool:workers-8core |
+| workers-32core |          8 | e2-standard-32 | pool:workers-32core                               |
+
+Some pools are labeled with `default-*-pool` labels. These labels specify which
+pool to use if it is not specified in the LoadTest configuration. With the
+configuration above, these tests (for instance, the tests specified in the
+[examples]) will use the `drivers` and `workers-8core` pools, and not interfere
+with continuous integration jobs. The default labels are defined as part of the
+controller build: if they are not set, the controller will only run tests where
+the `pool` labels are specified explicitly.
+
+## Controller deployment
+
+The steps for building and deploying a controller are described in the
+[deployment documentation].
+
+[deployment documentation]:
+  https://github.com/grpc/test-infra/blob/master/doc/deployment.md
+
+## Continuous integration
+
+Our continuous integration setup is described in the [gRPC OSS benchmarks
+README] in the [gRPC Core] repository. The main continuous integration job
+executes the script [grpc_e2e_performance_gke.sh] to generate the data presented
```

Comment:
Consider: "job executes ..." --> "job uses the [...] script to generate"

Reply:
Done.